Proposals news (#929)
akeneo/pim-helpcenter: content/md/products-super-power/review-products-proposal.md

To quickly find products with proposals, a search bar on product label and product identifier is available. Its behaviour is the same as in the [product grid](products-grid.html#search-by-label-and-identifier.html).

> The proposal grid also includes [filter management](products-grid.html#use-filters) to search on new attributes values. By default, the PIM displays the following filters: `family`, `author`, `source`, `proposed at` and `identifier`.

Reviewer: ```suggestion
The proposal grid also includes [filter management](products-grid.html#use-filters) to search on new attribute values. By default, the PIM displays the following filters: `family`, `author`, `source`, `proposed at`, and `identifier`.
```

![image](../img/Products_Proposals.png)

::: tips

> If you didn't receive a notification when the contributors sent to approve a proposal, check your [notifications preferences](manage-your-account.html#your-notifications-preferences-ee-only) in your user account. ;)

Reviewer: ```suggestion
If you didn't receive a notification when contributors have submitted a proposal for approval, check your [notifications preferences](manage-your-account.html#your-notifications-preferences-ee-only) in your user account. ;)
```

> | `Can be partially reviewed` | The proposal is complete and has been sent by the contributor. You have partial permissions to review the proposal changes | You can approve or reject only the changes on values on which you have an edit rights |

Reviewer: ```suggestion
| `Can be partially reviewed` | The proposal is complete and has been submitted by the contributor. You have partial permissions to review the proposal changes | You can approve or reject only the changes on values on which you have edit rights |
```

| `Can’t be reviewed`  | You have no permission to review the proposal changes | No action is available on the proposal                            |

![image](../img/Proposal_Status.png)

::: info

> If you have only view rights on some changes, the changes are displayed but you can do any action for this change.

Reviewer: ```suggestion
If you have only view rights on some changes, changes are displayed but you cannot do any action on these changes.
```

To approve a proposal:
1.  Go to the products’ proposals list, in the `Activity` menu for example

> 1.  Drag your mouse over a proposal having the statuses `Waiting for approval` or `Can be partially reviewed`. The proposal will be greyed out and the `Approve all` button will appear

Reviewer: ```suggestion
1.  Drag your mouse over a proposal having the status `Waiting for approval` or `Can be partially reviewed`. The proposal will be greyed out and the `Approve all` button will appear
```

To approve a change:
1.  Go to the products’ proposals list, in the `Activity` menu for example

> 1.  For the proposals with the `Waiting for approval` or `can be partially reviewed` status, if you have edit rights on the changes on values, a checkmark green button is displayed

Reviewer: ```suggestion
1.  For the proposals with the `Waiting for approval` or `can be partially reviewed` status, if you have edit rights on values changed, a checkmark green button is displayed
```
